Citizen-Science/src/screens/LoginScreen.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import { ImageBackground, SafeAreaView, Text, TouchableOpacity} from "react-native";
+import styles from '../StyleSheet';
+import CustomButton from '../components/CustomButton';
+import CustomTextField from '../components/CustomTextField';
+import { useNavigation } from '@react-navigation/native';
+import {NativeStackNavigationProp} from '@react-navigation/native-stack';
+import LogoName from "../components/LogoName";
+import BackButton from "../components/BackButton";
+import ScreenTitle from "../components/ScreenTitle";
+
+import {responsiveFontSize, responsiveHeight, responsiveWidth} from "react-native-responsive-dimensions";
+import Blobs from "../components/Blobs";
+
+
+type NavigationParam = {
+    Login: undefined;
+    Signup: undefined;
+    NavigationBar: undefined;
+};
+
+//Type for our Navigation in our component
+type NavigationProp = NativeStackNavigationProp<NavigationParam, 'Login'>;
+
+const LoginScreen: React.FC = () => {
+    const navigation = useNavigation<NavigationProp>();
+
+
+
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+
+    // handleLoginPress:
+    // Here we need to define what will happen when the login button is pressed. So user authentication.
+    const handleLoginPress = () => {
+        console.log('Login button pressed');
+        // logic for what should happen on login press
+        navigation.navigate('NavigationBar');
+    };
+
+    // @ts-ignore
```

Comment:
Do not use ts-ignore, defeats the purpose of using typescript. You may need to install the types library, or research the correct type

Reply:
I think we ended up resolving this, we originally used it to debug. But we'll make sure it works without it.